### PR TITLE
Settings page update and several other fixes

### PR DESCRIPTION
**Pull request by kyle-flynn**
Address #76

### CODE REVIEW

File: front-end/src/seasons/HydrogenHorizons/Settings.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+const SettingsTab: FC = () => {
+  const [motorOneOpenPosition, setMotorOneOpenPosition] = useRecoilState(
+    motorOneOpenPositionAtom
+  );
+  const [motorOneClosePosition, setMotorOneClosePosition] = useRecoilState(
+    motorOneClosePositionAtom
+  );
+  const [motorTwoOpenPosition, setMotorTwoOpenPosition] = useRecoilState(
+    motorOneOpenPositionAtom
```
Comment (NoahAndrews) [inline front-end/src/seasons/HydrogenHorizons/Settings.tsx:19]: Should be `motorTwoOpenPositionAtom`

File: front-end/src/seasons/HydrogenHorizons/Settings.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  );
+  const [motorTwoClosePosition, setMotorTwoClosePosition] = useRecoilState(
+    motorOneClosePositionAtom
```
Comment (NoahAndrews) [inline front-end/src/seasons/HydrogenHorizons/Settings.tsx:22]: Should be `motorTwoClosePositionAtom`

**Review by NoahAndrews — CHANGES_REQUESTED** [review]
Rather than using terms like "Motor One Open Position" and "Motor One Close Position", let's use "Red servo holding pulse width" and "Red servo released pulse width" (with adjusted settings names to match)

**Comment by kyle-flynn** [comment]
@NoahAndrews can you please re-review I've fixed the errors.

**Review by NoahAndrews — APPROVED** [review]
I'd also requested some renaming, but sure, we can go with this.

**Pull request merged**